src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -222,9 +223,12 @@ export function fetchFutureAppointments({ includeRequests = true } = {}) {
             }),
         );
       }
+
       const results = await Promise.all(promises);
-      const data = results[0]?.filter(appt => !appt.meta);
-      const backendSystemFailures = results[0]?.filter(appt => appt.meta);
+      const data = results[0]?.filter(appt => !appt.hasOwnProperty('meta'));
+      const backendServiceFailures = results[0]?.find(
+        appt => appt.hasOwnProperty('meta') || null,
+      );
```

Comment:
Modified to use 'find' since 'backendServiceFailures' is expected to be an Object